x-pack/plugins/observability_solution/observability_ai_assistant/server/functions/get_dataset_info/index.ts

```diff
@@ -48,7 +48,7 @@ export function registerGetDatasetInfoFunction({
 
       try {
         const body = await esClient.asCurrentUser.indices.resolveIndex({
-          name: index === '' ? '*' : index.split(','),
+          name: index === '' ? ['*', '*:*'] : index.split(','),
```

Comment:
This will only query remote indices if index is undefined.  Is that the goal?

I'll mention we've had issues query * remote clusters in the past with users wanting to specify which remote clusters because it can take a long time.